src/ibc_api/utils.py

```diff
@@ -381,18 +400,19 @@ def _download_file(src_file, dst_file, connector):
         return dst_file
     else:
         print(f"File {dst_file} already exists, skipping download.")
-
-        return []
+        return dst_file
 
 
-def download_data(db, save_to=None):
+def download_data(db, num_jobs=2, save_to=None):
```

Comment:
```suggestion
def download_data(db, n_jobs=2, save_to=None):
```
Just to match the joblib convention